theano/tensor/nnet/sigm.py

```diff
@@ -979,7 +987,9 @@ def local_1msigmoid(node):
             except Exception:
                 return
             if numpy.allclose(numpy.sum(val_l), 1):
-                return [sigmoid(-sub_r.owner.inputs[0])]
+                out = sigmoid(-sub_r.owner.inputs[0])
+                copy_stack_trace(node.outputs[0], out)
```

Comment:
I would tend to also add the stack trace of the initial sigmoid, `sub_r`.
I think it is OK not to add a stack trace to `-sub_r.owner.inputs[0]`.